frontend/src/app/shared/components/new-wdf-workplace-summary/wdf-workplace-summary.component.ts

```diff
@@ -253,10 +245,29 @@ export class WDFWorkplaceSummaryComponent implements OnInit, OnDestroy, OnChange
       'serviceUsers',
       'starters',
       'vacancies',
+      'capacities',
     ];
 
     return requiredFields.every(
       (field) => this.workplace.wdf[field].updatedSinceEffectiveDate || this.confirmedFields.includes(field),
     );
   }
+
+  protected setShowWdfConfirmations(): void {
+    Object.keys(this.showWdfConfirmations).forEach((field) => {
+      this.showWdfConfirmations[field] = this.showWdfConfirmation(field);
```

Comment:
At a first glance this might look a bit confusing due to [] and () similarity. Would it be ok if we rename the method that should return a boolean? 
```suggestion
      this.showWdfConfirmations[field] = this.shouldShowWdfConfirmation(field);
```